test/test_decomp.py

```diff
@@ -474,11 +471,11 @@ def check_decomposed(aten_name):
                 )
 
     def test_torchscriptable(self, device):
-        skip_list = []
+        skip_list = [aten.rsub.Scalar]
```

Comment:
how comeeee

Reply:
Since our decomposition is relying on implicit scalar => tensor promotion to work haha. But Torchscript has not overload for sub(scalar, tensor)